lib/mongo/server/connection_pool/queue.rb

```diff
@@ -165,12 +164,45 @@ def wait_timeout
           @wait_timeout ||= options[:wait_queue_timeout] || WAIT_TIMEOUT
         end
 
+        # The maximum seconds a socket can remain idle in the connection pool after it
+        #   has been checked out and checked back in.
```

Comment:
What Jesse said: "maximum seconds a socket can remain idle since it has been checked in to the pool."